xkcd/main.go

```diff
@@ -167,8 +171,14 @@ func Show(app fyne.App) {
 		}),
 		submit)
 	x.image = &canvas.Image{FillMode: canvas.ImageFillOriginal}
+	imageContainer := widget.NewScrollContainer(x.image)
+	controlsContainer := fyne.NewContainerWithLayout(
+		layout.NewBorderLayout(buttons, form, nil, nil),
+		buttons, form)
 	w.SetContent(fyne.NewContainerWithLayout(
-		layout.NewBorderLayout(form, buttons, nil, nil),
-		form, buttons, x.image))
+		layout.NewBorderLayout(controlsContainer, nil, nil, nil),
+		controlsContainer, imageContainer))
+	w.Resize(fyne.NewSize(1000, 800)) // will limit to screensize
```

Comment:
Are you sure this is a sensible start size? it seems very large?
I agree it will (depending on OS) likely limit to screen size, but it doesn't seem like the kind of window that wants to be so big.
Also I don't think we should be requesting center on screen without good reason.